zcash_test_vectors/unified_full_viewing_keys.py

```diff
@@ -100,6 +111,7 @@ def main():
             'unknown_fvk_typecode': unknown_tc,
```

Comment:
Please also include the seed, so that this can be used to test derivation the whole way through.

Reply:
Added https://github.com/daira/zcash-test-vectors/pull/1 for this.

Reply:
The seed is fixed (00 01 .. 1f) and doesn't change between test vector entries; do you want me to add it to all entries?